docs/src/tool/index.md

```diff
@@ -226,20 +236,39 @@ may be used in any JavaScript project like any other JavaScript file and library
 
 # {#ref-usage-devnet} `reach devnet`
 
-You can run a private Reach devnet by executing
+You can run a private Reach devnet by executing the following command:
 
 ```cmd
-$ reach devnet
+$ ./reach devnet
+```
+
+:::note
+If running `reach devnet`, it is recommended to permanently set `REACH_CONNECTOR_MODE` to the desired consensus network. If you did not set this when you originally ran `reach config`, then you can run `reach devnet` with the argument`REACH_CONNECTOR_MODE=[OPTION]` as in the following command.
+:::
+
+```cmd
+$ REACH_CONNECTOR_MODE=ALGO ./reach devnet
 ```
 
 `reach devnet` supports the following options:
 
-+ `--await-background` --- Run in background and await availability.
-+ The mandatory environment variable `REACH_CONNECTOR_MODE` specifies which devnet to run.
++ `--await-background` --- Run in the background and await availability.
+
+  The mandatory environment variable `REACH_CONNECTOR_MODE` specifies which devnet to run.
 The options are:
 
 + `ETH`, which runs an Ethereum devnet on `localhost:8545`
```

Comment:
These aren't right - `$CONN-live` is used when connecting to existing networks (like testnets or even the mainnet).

https://docs.reach.sh/tool/#p_44

For brevity's sake I think it'd be better to link to a consolidated section of the docs that outlines 'em in one place.

Reply:
I agree

Reply:
Ping on this